### PR TITLE
Feature - Migrate to GitBook

### PR DESCRIPTION
**Pull request by ZilvinasKucinskas**
* Migrate to https://docs.developer.fractal.id 
* Add `.gitbook.yaml` and customize `README.md` file to have a link to online documentation

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Table of contents
+
+* [Introduction](README.md)
```
Comment (hugopeixoto) [inline SUMMARY.md]: Should this be `INTRODUCTION.md`?

File: .gitbook.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+root: ./
+
+structure:
+  readme: INTRODUCTION.md
```
Comment (hugopeixoto) [inline .gitbook.yaml]: Why are these two files all upper case?

File: SUMMARY.md (hunk 1 above)
Comment (ZilvinasKucinskas) [reply SUMMARY.md]: Yes, thanks for noticing. I will try syncing the branch first to check how it looks like before the merging and syncing `master`.

File: .gitbook.yaml (hunk 2 above)
Comment (ZilvinasKucinskas) [reply .gitbook.yaml]: GitBook generates these 2 main files in uppercase. No other reason. With .gitbook.yaml we can probably rename them to lowercase.

Comment (hugopeixoto) [reply .gitbook.yaml]: You may have some troubles lower casing the actual files, if you use osx.

Comment (ZilvinasKucinskas) [reply .gitbook.yaml]: Ran into this already 😅 , had to:

```
$ git mv somename tmpname
$ git mv tmpname SomeName
```

**Pull request merged**